app/templates/ui/app/search/search.controller.js

```diff
@@ -22,5 +22,11 @@
     $scope.$watch(userService.currentUser, function(newValue) {
       ctrl.currentUser = newValue;
     });
+
+    $scope.setSnippet = function(type) {
```

Comment:
our convention has been to put functions on the ctrl variable:

ctrl.setSnippet = ....

see below for the html approach